src/components/Footer/index.tsx

```diff
@@ -7,13 +7,10 @@ import {
 import { Link } from "react-router-dom";
 import classes from "./index.module.css";
 
-const data = [
+const allData = [
```

Comment:
try another name ?

Reply:
I should take a look at this later with VSCode

Reply:
i have no idea how to name the list

Reply:
> i have no idea how to name the list

try to rename that to make that more readable pls

Reply:
> > i have no idea how to name the list
> 
> try to rename that to make that more readable pls

change from `allData` to `contentData` cuz each (list of) data is used locally for that file only (that is, the content of the component only) so i just call it with a general name like that

Reply:
broken english boi demanding different variable name from boi living in US